components/HackneyDS/Collapse/Collapse.js

```diff
@@ -9,9 +9,11 @@ export const Collapse = ({
   title,
   className = '',
   expanded,
+  triggerOnlyButton,
```

Comment:
prop name doesnt make sense

Reply:
how i should name it?

Reply:
well what does it do?

Reply:
![image](https://user-images.githubusercontent.com/77606518/141430811-f23264f9-6db8-4d05-9471-0064fc5b276b.png)
For now, when you click on full container, it (expand, collapse) content. If you want, that (expand, collapse) work only for collapse button, you can use this boolean value
![image](https://user-images.githubusercontent.com/77606518/141431084-3d0e526e-ca4a-43ab-9945-f7b1b7061efc.png)

Reply:
Okay, maybe something like buttonPointerEventsOnly

Reply:
isButtonClickOnly